src/plugins/query_enhancements/public/query_assist/utils/get_is_summary_agent.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpSetup } from 'opensearch-dashboards/public';
+import { API } from '../../../common/constants';
+
+export async function checkAgentsExist(
+  http: HttpSetup,
+  agentConfigName: string | string[],
+  dataSourceId?: string
+) {
+  const queryParams = new URLSearchParams();
+  if (Array.isArray(agentConfigName)) {
+    agentConfigName.forEach((name) => queryParams.append('agentConfigName', name));
+  } else {
+    queryParams.append('agentConfigName', agentConfigName);
+  }
+  queryParams.append('dataSourceId', dataSourceId);
+
+  const response = await http.get(API.AGENT_API.CONFIG_EXISTS, {
+    query: Object.fromEntries(queryParams),
```

Comment:
There might be problem to call `Object.fromEntries(queryParams)`, for example:
```typescript
// if calling append two times
queryParams.append('agentConfigName', 'name1')
queryParams.append('agentConfigName', 'name2')
// this returns {agentConfigName: 'name2'}, which name1 is missing
Object.fromEntries(queryParams),
```
I think you can simply call in this way
```
  const response = await http.get(API.AGENT_API.CONFIG_EXISTS, {
    query: { agentConfigName, dataSourceId },
  });
```

Reply:
thanks for the advice，already updated~